ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/oauth/OAuthServiceClientImpl.java

```diff
@@ -8,18 +8,21 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.ide.api.auth;
+package org.eclipse.che.ide.oauth;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
 
 import org.eclipse.che.api.auth.shared.dto.OAuthToken;
+import org.eclipse.che.ide.api.oauth.OAuthServiceClient;
 import org.eclipse.che.ide.rest.AsyncRequestCallback;
 import org.eclipse.che.ide.rest.AsyncRequestFactory;
 import org.eclipse.che.ide.rest.RestContext;
 
-import javax.inject.Inject;
-
 /**
  * @author Sergii Leschenko
  */
+@Singleton
```

Comment:
Why did you add this annotation?
It's already bound as singleton in CoreGinModule.